lib/content-services/src/lib/search/components/search-filter-tabbed/search-filter-tabbed.component.scss

```diff
@@ -0,0 +1,10 @@
+adf-search-filter-tabbed {
+    .mat-tab-label {
+        flex: 1;
+    }
+
+    .mat-tab-body.mat-tab-body-active .mat-tab-body-content {
+        display: block;
+        visibility: visible !important;
```

Comment:
Can you remove !important? It should not be needed if you have encapsulation set to None

Reply:
This is a bug in angular material when using MatTabGroup in conjunction with MatMenu, where the `visibility: hidden` inline styling does not get removed properly. I've reported the bug [here](https://github.com/angular/components/issues/27426), and added a comment on this css code, explaining the same